pymeasure/instruments/cisystems/sr200n.py

```diff
@@ -0,0 +1,181 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2023 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+import struct
+from pymeasure.instruments import Instrument
+from pymeasure.instruments.common_base import CommonBase
+
+
+# - PARAMETER STRINGS - #
+TEMP = '0817bf'
+TEMP_STABLE = '0817ca'
+SET_POINT_MIN = '0817bc'
+SET_POINT_MAX = '0817bd'
+SET_POINT_GET = '0817c2'
+SET_POINT_SET = '0617be %f'
+CHOPPER_FREQ_GET = '081939'
+CHOPPER_FREQ_SET = '06193a %f'
+CHOPPER_POS = '06193b %i'
+
+
+class SR200N(Instrument):
+
+    # - temperature parameters ------------------------------ #
+    temp = CommonBase.measurement(
```

Comment:
Please use the Instrument's measurement etc.
```suggestion
    temp = Instrument.measurement(
```